lib/events/message.js

```diff
@@ -16,6 +20,21 @@ module.exports = (data, cb) => {
   // }).catch(function (e) {
   //   cb(e)
   // })
+  if (data.event.text.includes('roll')) {
```

Comment:
My thinking right now is that this should check if the channel's purpose includes the string `[gr]` or some other keyword, and otherwise not do anything. Otherwise, it might be nice to just check the channel _name_ for `gender` as a prefix (so `genderqueer` and `genderfuck` would both work), and assume that'll DWIM. I just don't think it's good to have this turned on for all channels, mostly cause there's a shared instance for several non-queer slacks.